webapp/layouts/default.vue

```diff
@@ -217,65 +17,28 @@
 </template>
 
 <script>
-import { mapGetters } from 'vuex'
-import LOGOS from '../constants/logos.js'
-import headerMenu from '../constants/headerMenu.js'
+import HeaderMenu from '~/components/HeaderMenu/HeaderMenu'
 import seo from '~/mixins/seo'
-import AvatarMenu from '~/components/AvatarMenu/AvatarMenu'
-import FilterMenu from '~/components/FilterMenu/FilterMenu.vue'
-import InviteButton from '~/components/InviteButton/InviteButton'
-import LocaleSwitch from '~/components/LocaleSwitch/LocaleSwitch'
-import Logo from '~/components/Logo/Logo'
-import SearchField from '~/components/features/SearchField/SearchField.vue'
 import Modal from '~/components/Modal'
-import NotificationMenu from '~/components/NotificationMenu/NotificationMenu'
 import PageFooter from '~/components/PageFooter/PageFooter'
-import links from '~/constants/links.js'
-import PageParamsLink from '~/components/_new/features/PageParamsLink/PageParamsLink.vue'
 
 export default {
   components: {
-    AvatarMenu,
-    FilterMenu,
-    InviteButton,
-    LocaleSwitch,
-    Logo,
+    HeaderMenu,
     Modal,
-    NotificationMenu,
     PageFooter,
-    PageParamsLink,
-    SearchField,
   },
   mixins: [seo],
   data() {
     return {
       windowWidth: null,
-      links,
-      LOGOS,
-      isHeaderMenu: headerMenu.MENU.length > 0,
-      menu: headerMenu.MENU,
-      mobileSearchVisible: false,
-      toggleMobileMenu: false,
-      inviteRegistration: this.$env.INVITE_REGISTRATION === true, // for 'false' in .env INVITE_REGISTRATION is of type undefined and not(!) boolean false, because of internal handling,
-      categoriesActive: this.$env.CATEGORIES_ACTIVE,
+      maxMobileWidth: 811,
     }
   },
   computed: {
-    ...mapGetters({
-      isLoggedIn: 'auth/isLoggedIn',
-    }),
-    showFilterMenuDropdown() {
-      const [firstRoute] = this.$route.matched
-      return firstRoute && firstRoute.name === 'index'
-    },
     showMobileMenu() {
       if (!this.windowWidth) return false
-      return this.windowWidth < 810
-    },
-  },
-  methods: {
-    toggleMobileMenuView() {
-      this.toggleMobileMenu = !this.toggleMobileMenu
+      return this.windowWidth < this.maxMobileWidth
```

Comment:
```suggestion
      return this.windowWidth <= this.maxMobileWidth
```

That fits better to the name `maxMobileWidth` and the logic is more consitent if we use `810px` all over the place and not `811px`.